test/controllers/manufacturing/AluminiumOccupationList1ControllerSpec.scala

```diff
@@ -22,22 +22,36 @@ import controllers.actions.UnAuthed
 import forms.manufacturing.AluminiumOccupationList1FormProvider
 import models.{NormalMode, UserAnswers}
 import navigation.{FakeNavigator, Navigator}
+import org.mockito.Matchers._
+import org.mockito.Mockito._
 import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.mockito.MockitoSugar
 import pages.ClaimAmount
 import pages.manufacturing.AluminiumOccupationList1Page
+import play.api.Application
 import play.api.inject.bind
 import play.api.mvc.Call
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 import repositories.SessionRepository
 import views.html.manufacturing.AluminiumOccupationList1View
 
-class AluminiumOccupationList1ControllerSpec extends SpecBase with ScalaFutures with IntegrationPatience {
+import scala.concurrent.Future
+
+class AluminiumOccupationList1ControllerSpec extends SpecBase with ScalaFutures with MockitoSugar with IntegrationPatience {
 
   def onwardRoute = Call("GET", "/foo")
 
-  val formProvider = new AluminiumOccupationList1FormProvider()
-  val form = formProvider()
+  private val formProvider = new AluminiumOccupationList1FormProvider()
+  private val form = formProvider()
+  private val userAnswers = emptyUserAnswers
+  private val mockSessionRepository = mock[SessionRepository]
+
+  when(mockSessionRepository.set(any(), any())) thenReturn Future.successful(true)
+
+  def sessionApplication: Application = applicationBuilder(userAnswers = Some(userAnswers))
```

Comment:
Why did we suddenly change pattern. All other implementation follows the same process, I don't see why this one should be different. Either do this for all or keep this inline with all other test files.